src/instruction_interpreter.py

```diff
@@ -258,6 +258,19 @@ def interpret_group_F(self, instruction):
         elif instruction & 0xFF == 0x18:
             # Fx18 - LD ST, V
             self.reg_sound = self.reg_v[x]
+        elif instruction & 0xFF == 0x33:
+            '''
+            Fx33 - LD B, Vx
+            Store BCD representation of Vx in memory locations I, I+1, and I+2.
+
+            The interpreter takes the decimal value of Vx, and places the
+            hundreds digit in memory at location in I, the tens digit at
+            location I+1, and the ones digit at location I+2.
+            '''
+            self.memory[self.reg_i] = int(str(self.reg_v[x])[0])  # Hundreds digit
+            self.memory[self.reg_i + 1] = int(str(self.reg_v[x])[1])  # Tens digit
+            self.memory[self.reg_i + 2] = int(str(self.reg_v[x])[2])  # Ones digit
+
```

Comment:
IndexError: string index out of range, if reg_v[x] is less than 100.
Instead of you can do something like
```
digits = format(self.reg_v[x], "03d")
self.memory[self.reg_i] = int(digits[0])
...
```